xbmc/input/linux/LinuxInputDevices.h

```diff
@@ -87,9 +86,9 @@ class CLinuxInputDevice
   bool m_bUnplugged;
   std::deque<XBMC_Event> m_equeue;
   int m_mt_currentSlot;
-  int m_mt_x[TOUCH_MAX_POINTERS];
-  int m_mt_y[TOUCH_MAX_POINTERS];
-  TouchInput m_mt_event[TOUCH_MAX_POINTERS];
+  int m_mt_x[CGenericTouchInputHandler::MAX_POINTERS];
```

Comment:
If we're modernizing it would make sense to make these `std::aray` but might be out of scope for this PR.